components/Profile/Business/BusinessAPI.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { API, Auth, graphqlOperation, Storage } from "aws-amplify";
+import { createBusiness, updateBusiness } from "../../../src/graphql/mutations";
+import { Business } from "../../../src/API";
+
+export default class BusinessAPI {
+  static async create(business: Business) {
+    const user = await Auth.currentAuthenticatedUser();
+    const emailAttribute = user.attributes.email;
+
+    const businessObj = { ...business, email: emailAttribute };
```

Comment:
the User object has a "businesses" 1:n relationship, I think it's done like an array of pointers. This allows for the user to have ownership of multiple businesses, and can be easily done by setting the "userID" field in the Business object to that of the user who owns it (userID is found in the redux User object).

So you could add {...business, email: emailAttribute, userID: <ID from user in redux> } and that should allow for ownership

Reply:
Agreed. Is the email category supposed to refer to the owner's email or the business's email? If it's the former, it's kind of redundant since we already have a pointer the the owner's User data, which includes their email.

Reply:
Maybe the owner's email would be different than the front-facing business contact email. Right now we don't have functionality for editing/updating emails so it's just using the owner's email for now.

I updated it to use the user in redux instead of the async call